target/cheri-common/cheri_defs.h

```diff
@@ -154,7 +154,16 @@ typedef enum CheriTbFlags {
      * PCC spans the full adddress space and has base zero. This means we do
      * not need to perform bounds checks or subtract/add PCC.base
      */
-    TB_FLAG_PCC_FULL_AS = (1 << 7)
+
+    TB_FLAG_PCC_FULL_AS = (1 << 7),
```

Comment:
Since this PR basically touches everywhere using this, can we rename it to TB_FLAG_CHERI_PCC_FULL_AS for consistency?